core/src/main/scala/org/apache/spark/rdd/HadoopRDD.scala

```diff
@@ -196,6 +197,24 @@ class HadoopRDD[K, V](
     newInputFormat
   }
 
+  private val UNSPLITTABLE_FILE_SIZE_LOG_THRESHOLD = 1024 * 1024 * 1024
```

Comment:
can we make it configurable?

Reply:
Add a config `spark.io.file.unsplittable.warning.threshold`